eclair-core/src/main/scala/fr/acinq/eclair/payment/send/MultiPartPaymentLifecycle.scala

```diff
@@ -220,6 +236,8 @@ class MultiPartPaymentLifecycle(nodeParams: NodeParams, cfg: SendPaymentConfig,
 
 object MultiPartPaymentLifecycle {
 
+  val parentPaymentIdKey = Context.key[UUID]("parentPaymentId", UUID.fromString("00000000-0000-0000-0000-000000000000"))
```

Comment:
What's this 0 UUID? Do we need to keep a map of static assigned UUIDs somewhere?

Reply:
Kamon just needs a default key for some reason, it doesn't really matter a lot because we set it when we need it.